_episodes_rmd/02-raster-plot.Rmd

```diff
@@ -139,7 +131,7 @@ Since we have three bins, we want to use the first three terrain colors:
 terrain.colors(3)
 ```
 
-We can see the `terrain.colors()` function returns *hex colors* - 
+The `terrain.colors()` function returns *hex colors* - 
```

Comment:
"much like the `rgb()` function used in episode 1" perhaps?

Reply:
Ah, good point! We should add a reminder to the learners that they've seen this before! I'll put in an issue.

Reply:
https://github.com/datacarpentry/r-raster-vector-geospatial/issues/195